docs/extensions/index.md

```diff
@@ -0,0 +1,121 @@
+<sub>This image was created using <a href="https://github.com/sayanshaw24/combine" target="_blank">Combine.AI</a>, which is powered by Bing Chat, Bing Image Creator, and EdgeGPT.</sub>
+
+### Quickstart
+
+## Installation
+### **Python installation**
+```bash
+pip install onnxruntime-extensions
+````
+
+
+### **Nightly Build**
+
+#### <strong>on Windows</strong>
+```cmd
+pip install --index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ onnxruntime-extensions
+```
+Please ensure that you have met the prerequisites of onnxruntime-extensions (e.g., onnx and onnxruntime) in your Python environment.
+#### <strong>on Linux/macOS</strong>
+Please make sure the compiler toolkit like gcc(later than g++ 8.0) or clang are installed before the following command
+```bash
+python -m pip install git+https://github.com/microsoft/onnxruntime-extensions.git
+```
+
+## Prepare the pre/post-processing ONNX model
+With onnxruntime-extensions Python package, you can easily get the ONNX processing graph by converting them from Huggingface transformer data processing classes such as in the following example:
+```python
+import onnxruntime as _ort
+from transformers import AutoTokenizer
+from onnxruntime_extensions import OrtPyFunction, gen_processing_models
+
+tokenizer = AutoTokenizer.from_pretrained('distilbert-base-uncased')
+model = OrtPyFunction(gen_processing_models(tokenizer, pre_kwargs={})[0])
+```
+
+For more information, you can check API using the following:
+```python
+help(onnxruntime_extensions.gen_processing_models)
+```
+### NOTE: These data processing model can be merged into other model [onnx.compose](https://onnx.ai/onnx/api/compose.html) if needed.
+## Inference with ONNX Runtime Extensions
+
+### Python
+There are individual packages for the following languages, please install it for the build.
+```python
+import onnxruntime as _ort
+from onnxruntime_extensions import get_library_path as _lib_path
+
+so = _ort.SessionOptions()
+so.register_custom_ops_library(_lib_path())
+
+# Run the ONNXRuntime Session as per ONNXRuntime docs suggestions.
+sess = _ort.InferenceSession(model, so)
+sess.run (...)
+```
+### C++
+
+```c++
+  // The line loads the customop library into ONNXRuntime engine to load the ONNX model with the custom op
+  Ort::ThrowOnError(Ort::GetApi().RegisterCustomOpsLibrary((OrtSessionOptions*)session_options, custom_op_library_filename, &handle));
+
+  // The regular ONNXRuntime invoking to run the model.
+  Ort::Session session(env, model_uri, session_options);
+  RunSession(session, inputs, outputs);
+```
+### Java
+```java
+var env = OrtEnvironment.getEnvironment();
+var sess_opt = new OrtSession.SessionOptions();
+
+/* Register the custom ops from onnxruntime-extensions */
+sess_opt.registerCustomOpLibrary(OrtxPackage.getLibraryPath());
+```
+
+### C#
+```C#
```

Comment:
C# formatting not working

Reply:
likely a jekyll issue because it works in the md preview - changed it to java as the styling is the same